src/Utility/browser.ts

```diff
@@ -89,3 +91,17 @@ export const clientPost = (
     form.submit();
     document.body.removeChild(form);
 };
+
+// This uses the `navigator.language` property (similar-ish to the `Accept-Language` header which we cannot access from
+// JS) which may not necessarily represent the user's country (or even include region information at all).
+// The more reliable way would be to feed the user's IP into a geolocation service but that is not an option, so we have
+// to stick with this.
+export const guessUserCountry = (): Country => {
+    const { country } = parseBcp47Tag(navigator.language) as { country: Country };
```

Comment:
The type annotation should really be a proper return type annotation on the function…

Reply:
But it isn't guaranteed that `parseBcp47Tag()` always results in something that satisfies our `Country` type. I think it's better to be aware of that in downstream usages and cast there when we consider it appropriate.

Reply:
I am not completely sold on this… Firstly, the cast here should definitely include `undefined` (I know this changes nothing, but casting a potentially undefined value to `Country` is wrong and could lead to bugs in the future). Secondly, for this function to be safe, this requires proper type checking and maybe a typeguard. I can see how we do not need in in our case (although I think the more elegant solution would be to define `window.SUPPORTED_COUNTRIES.includes(country)` as a typeguard), but in other cases it might. I can see how this is something that can happen downstream.